### DISCUSSION TITLE
It's possible to add checkbox

### DISCUSSION THREAD

**Discussion by komxvch** [sagalbot/vue-select#1515] [Q&A]
Hi there.
Thank you, for this library.
Could you help me please, how I can to add checkboxes in select list items, and how I can to see checked items?

**Comment by komxvch** [comment]
The answer to my first question is to use` <slot>`


```
<template slot="option" slot-scope="option">
        <input type="checkbox" style="height:10px; width:10px;background-color: initial;
    cursor: default;
    appearance: auto;
    box-sizing: border-box;
    margin: 3px 3px 3px 4px;
    padding: initial;
    border: initial;">
        <span>{{option.label}}</span>
    </template>
```

But the second part is open about checked items

**Comment by wdsmaduranga** [comment]
how to add checkbox

**Comment by wdsmaduranga** [comment]
> <template slot="option" slot-scope="option">
>         <input type="checkbox" style="height:10px; width:10px;background-color: initial;
>     cursor: default;
>     appearance: auto;
>     box-sizing: border-box;
>     margin: 3px 3px 3px 4px;
>     padding: initial;
>     border: initial;">
>         <span>{{option.label}}</span>
>     </template>

click and not tics
